setup.py

```diff
@@ -28,6 +28,7 @@
                  version=version,
                  packages=setuptools.find_packages(),
                  package_data={'': ['*.txt', '*.md']},
+                 python_requires=">= 3.5",
```

Comment:
I think the space before the 3.5 shouldn't be there.

Reply:
My bad, fixed.